### ISSUE TITLE
BibleTime is not starting on windows 7

### ISSUE THREAD

**Issue by antskarner** [bibletime/bibletime#90]
I do not know is this a bug or not, but when I start Bibletime as usual, nothing happens Then I started it via command prompt with --debug. An here is results: `(BibleTime 2.11.0+52ef4a4d7f0d0902a0d90d6ef60c07da70b89ef7) WARNING: Could not create user settings directory relative to "C:/Program Files/BibleTime/bin"
(BibleTime 2.11.0+52ef4a4d7f0d0902a0d90d6ef60c07da70b89ef7) FATAL: Error initializing directory cache!
Please report this bug at https://github.com/bibletime/bibletime/issues
` Ih have no programming skills, but I hope that info eikk make this excellent brogram eve better. God bless you!

**Comment by antskarner** [comment]
Oh, I forgot to add info about Bibletime version. It is 2.11.0. Sorry for that!

**Comment by antskarner** [comment]
I found a solution for this problem: I had to run Bibletime as admistrator. And program started as it should normally.  Then I ran it normally - everything works!

**Comment by gholmlund** [comment]
The HOME directory for settings is defined in Windows by the value of the AppData environment variable. Please do "echo %AppData% in the command window and post its value.

**Comment by antskarner** [comment]
Here it is:
`echo %AppData%
C:\Users\Ats\AppData\Roaming` Now I can add Bibles, etc only when I run the program in administrator mode.

**Comment by gholmlund** [comment]
I assume the AppData value is for being logged into the non-administrator?

**Comment by gholmlund** [comment]
Never mind the previous request. I have found the problem. It is a Windows specific problem.

Technical details:
A defect on Windows related to a HOME directory containing international chars was fixed on Nov 23, 2016. This involved changing calls to getenv and putenv to GetEnvironmentVariable and PutEnvironmentVariable. This is in main.cpp. This is necessary to do before Sword is initialized because it uses $HOME for its settings and module storage. 

However in utils/directory.cpp a qgetenv is used to read the $HOME variable. It appears to be cached and its value does not change due to the code in main.cpp. To fix this issue the qgetenv must be changed to a GetEnvironmentVariable. (for Windows)
